lib/provisioner/worker/plugins/providers/fog_provider/lib/fog_provider/aws.rb

```diff
@@ -130,6 +130,9 @@ def confirm(inputmap)
         'bind_v4' => bind_ip
       }
       @result['hostname'] = hostname
+      @result['keys'] = {
```

Comment:
can we use something more descriptive than ``keys``?  maybe even ``['ssh']['host_keys']``.  we've so many kinds of keys already...